### PR TITLE
Add Activity#activity_target support

### PR DESCRIPTION
**Pull request by kennym**

### CODE REVIEW

File: lib/stream_rails/enrich.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def track_not_enriched_field(field, value = nil)
 
   class Enrich
     def initialize(fields = nil)
-      @fields = fields || [:actor, :object]
+      @fields = fields || [:actor, :object, :target]
```
Comment (tbarbugli) [inline lib/stream_rails/enrich.rb]: @kennym a bit OT in other libraries we auto detect references to enrich, can you create an issue for that?

Comment (kennym) [inline lib/stream_rails/enrich.rb]: @tbarbugli makes sense. Done: https://github.com/GetStream/stream-rails/issues/47

**Comment by kennym** [comment]
@tbarbugli do you think this PR makes sense?

**Comment by kennym** [comment]
plan-fnnjqwMFfeuOc

**Pull request merged**